services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -195,3 +303,104 @@ def produce_map_01_credential_payload(cls, did: str, permit_amendment: PermitAme
+                base.Identifier(
+                    scheme=ANONCRED_SCHEME,
+                    identifierValue=str(orgbook_entity.registration_id),
+                    identifierURI=orgbook_cred_url)
+            ])
+
+        facility = cc.Facility(
+            name=permit_amendment.mine.mine_name,
+            geolocation=
+            f'https://plus.codes/{plus_code_encode(permit_amendment.mine.latitude, permit_amendment.mine.longitude)}',
+            verifiedByCAB=True)
+
+        products = [
+            cc.Product(
+                name=c,
+                classifications=cc.Classification(
+                    scheme="https://unstats.un.org/unsd/classifications/Econ/cpc",
+                    classifierName=c),
+                verifiedByCAB=False) for c in permit_amendment.mine.commodities
+        ]
+
+        untp_assessments = [
+            cc.ConformityAssessment(
+                referenceRegulation=cc.Regulation(
+                    id="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01",
+                    name="BC Mines Act",
+                    issuingBody=base.Party(name="BC Government"),
+                    effectiveDate=datetime(2024, 5, 14, tzinfo=ZoneInfo("UTC")).isoformat()),
+                                                                                                   # Is there a did:web that attests to that legistlation?
+                subjectFacilities=[facility],
+                subjectProducts=products,
+                sustainabilityTopic=codes.SustainabilityTopic.Governance_Compliance)
+        ]
+        issue_date = permit_amendment.issue_date
+        issuance_date_str = datetime(
+            issue_date.year, issue_date.month, issue_date.day, 0, 0, 0,
+            tzinfo=ZoneInfo("UTC")).isoformat()
+
+        cred = cc.ConformityAttestation(
+            id="http://example.com/attestation/123",
+            assessmentLevel=codes.AssessmentAssuranceCode.GovtApproval,
+            type=codes.AttestationType.Certification,
+            description=
+            "This is a conformity attestation for the existence and good standing of a mining permit under the Mines Act within British Columbia (a province of Canada).",
+            scope=cc.ConformityAssessmentScheme(
+                id=
+                "https://github.com/bcgov/bc-vcpedia/blob/main/credentials/bc-mines-act-permit/1.1.1/governance.md",
+                name="BC Mines Act Permit Credential (1.1.1) Governance Documentation"),
+            issuedBy=untp_party_cpo,
+            issuedTo=untp_party_business,
+            validFrom=issuance_date_str,                                                                                                                                   #shouldn't this just be in the w3c wrapper
+            assessments=untp_assessments)
+
+        w3c_cred = W3CCred(
+            context=["https://www.w3.org/2018/credentials/v1", {
+                "@vocab": "urn:bcgov:attributes#"
```

Comment:
Lets start thinking about a context file/vocab for the mines stuff

Reply:
true... next PR. 

I'll put that in the backlog.